crates/core/src/node/inner/blockchain.rs

```diff
@@ -487,11 +496,12 @@ impl Blockchain {
                     fork_details.api_block.clone(),
                 )]),
                 hashes: HashMap::from_iter([(fork_details.block_number, fork_details.block_hash)]),
+                // Batches are not being used when running in forking mode
+                batches: HashMap::from_iter([]),
```

Comment:
Why would batches not be used during forking?

Reply:
Because we do not support L1-L2 communication in forking mode and that's the only reason we persist batches starting from this PR. I have made it a bit more clear and added a CLI options check (it's not pretty but should do for now).